src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,114 @@
+package duke.parser;
+
+import duke.command.AddCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.DoneCommand;
+import duke.command.ExceptionCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.command.UnknownCommand;
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * Represents the parser that parses user input to make sense of the given input and produce
+ * the appropriate command.
+ */
+public class Parser {
+    /**
+     * Takes in a String input and parses it to produce the appropriate command
+     * @param input the user input.
+     * @return the appropriate command.
+     */
+    public static Command parse(String input) {
+        try {
+            String[] words = input.split(" ", 2);
+            String keyWord = words[0];
+            String rest;
+            String[] splitRest;
```

Comment:
Perhaps change the naming to a term that could be less confused as a verb such as `restOfString`?